Library/Homebrew/os/mac/sdk.rb

```diff
@@ -25,14 +25,44 @@ class BaseSDKLocator
       class NoSDKError < StandardError; end
 
       def sdk_for(v)
-        path = sdk_paths[v]
-        raise NoSDKError if path.nil?
+        sdk = all_sdks.find { |s| s.version == v }
+        raise NoSDKError if sdk.nil?
 
-        SDK.new v, path, source
+        sdk
       end
 
       def all_sdks
-        sdk_paths.map { |v, p| SDK.new v, p, source }
+        @all_sdks ||= begin
+          # Bail out if there is no SDK prefix at all
+          if File.directory? sdk_prefix
+            sdks = []
+
+            Dir[File.join(sdk_prefix, "MacOSX*.sdk")].each do |sdk_path|
```

Comment:
No need to use `File.join` here, just hardcode `/`

Reply:
Yeah, this is actually just old code (GitHub doesn't show the diff very well). I'll change it though.